src/dev/amdgpu/AMDGPU.py

```diff
@@ -81,8 +81,8 @@ class AMDGPUDevice(PciDevice):
     InterruptPin = 2
     ExpansionROM = 0
 
-    rom_binary = Param.String("ROM binary dumped from hardware")
-    trace_file = Param.String("MMIO trace collected on hardware")
+    rom_binary = Param.String("", "Deprecated - This is ignored")
```

Comment:
Should we just remove these lines completely instead of removing the contents?

Reply:
Yea, Let me change it to do that.